src/components/Article/index.tsx

```diff
@@ -11,8 +11,13 @@ export const Article = ({ article, onCommentsClick }: ArticleProps) => {
   return (
     <div className="h-dyn-screen snap-center flex flex-col justify-end items-center relative max-w-[500px] mx-auto">
       <Image article={article} />
-      <div className="from-black/50 to-transparent bg-gradient-to-t p-3 pb-8 max-w-screen">
-        <div className="w-[calc(100%-0.75rem)] max-w-screen grid grid-cols-[90%_10%] gap-3 box-">
+      <div className={`
+          ${article.cover_image && "justify-end"}
+          ${!article.cover_image && "justify-center"}
+          relative p-3 pb-8 max-w-screen h-dyn-screen flex flex-col items-center
+          before:content-[""] before:w-full before:h-full before:absolute before:top-0 before:left-0 before:from-black/50 before:to-transparent before:bg-gradient-to-t before:blur-md`
```

Comment:
It looks very beautiful with this gradient, I liked the idea!
However doing this using a pseudo-element seems dirty, I believe the cleanest place to do this kind would actually be on the `Image` component, as the else of the ternary I mentioned.